webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require("path");
+
+const config = {
+  entry: {
+    ["latest-news"]: "./static/js/latest-news.js",
+    ["global-nav"]: "./static/js/global-nav.js"
+  },
+  output: {
+    path: path.resolve(__dirname, "static/dist"),
+    filename: "[name].js"
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js?$/,
+        exclude: /(node_modules)/,
```

Comment:
Does `global-nav` work with this config?
I recall that if global nav is imported as a module it uses ES6 syntax and requires babel transpilation.
For build.snapcraft.io I needed to do more complicated exclude rules to make it work: https://github.com/canonical-websites/build.snapcraft.io/blob/master/webpack/loaders-config.js#L6-L10

Reply:
It does seem to work here, not sure why.

I just notice there is difference in version. Latest global nav is now 2.0.5 (and it's `@canonical-webteam/global-nav` now), you are using older 2.0.2 which may not use ES6 modules yet.

Reply:
Ahh, I updated it to the latest version now. It still all seems to run without any exclude rules.

Reply:
Upgraded the version. Still seems to work fine with the current config.